app/models/forms/affiliation.rb

```diff
@@ -0,0 +1,41 @@
+module Forms
+  class Affiliation < Base
+    # Indicates that this is a deposit, and therefore should be fully validated.
+    attr_accessor :_deposit
+    attr_accessor :id
+    attr_accessor :label
+    attr_accessor :uri
+    attr_accessor :department
+    attr_accessor :abstract_contributor
+    attr_accessor :_destroy
+
+    validates :label, presence: true, if: :_deposit
+
+    def main_model
+      affiliation
+    end
+
+    def affiliation
+      @affiliation ||= begin
+        affiliation = if id.present?
```

Comment:
Update an existing affiliation if there is one; otherwise populate a new affiliation.